pyproject.toml

```diff
@@ -28,7 +28,7 @@ typing-extensions = ">=4.0.0, <4.6.0"
 pyjwt = ">=2.0.1, <3.0.0"
 scipy = "*"
 fastapi-vite = "^0.3.1"
-cai-causal-graph = ">=0.0.1"
+cai-causal-graph = ">=0.1.0"
```

Comment:
Wasn't sure if this was necessary or built automatically as union of each package toml.

Reply:
Yes, this is the combined dependency range so needs to be updated if any of the packages change their deps. This can be done automatically with `borg install` as well. 

All looks good, will merge once CI passes.